packages/lib/services/interop/InteropService.ts

```diff
@@ -58,6 +58,14 @@ export default class InteropService {
 					isNoteArchive: false, // Tells whether the file can contain multiple notes (eg. Enex or Jex format)
 					description: _('Markdown'),
 				},
+				{
+					...defaultImportExportModule(ModuleType.Importer),
+					format: 'yaml',
```

Comment:
Shouldn't that be "md" too? Also how does it work with the CLI tool - is it possible to disambiguate and select one format or the other (regular MD or MD+YAML)? I remember having the same issue with "ENEX as MD" vs "ENEX as HTML".

Reply:
My understanding is that this format is just a short name for the module. So for the CLI tool, you could use `--format yaml`. Although based on your other comment this will need to be changed, I'm thinking `md-frontmatter`.
How did you solve the enex issue? Based on the cli docs, it looks like it still exists.

Reply:
Right, I got confused. For ENEX, what I had done is add an "output" parameter to the import command, so you can import an ENEX file and set the output to create notes in either MD or HTML format.

But I guess that wouldn't apply in your case since you always create Markdown notes? But actually that makes me wonder - you've added html as supported format. In that case, how do you handle it? I'm not seeing any code that would process html files?

And for the format in general, you're right, it just need to be a unique identifier. "md-frontmatter" would indeed make sense.

Reply:
I ended up going with `md_frontmatter` because it works better with JavaScript.

The HTML files are html in file extension only (this is the same for the Md export). It's a way of indicating to the user (and any system that ingests the export) that the file contains html and not markdown. But i still wanted the file to be renderable by a markdown editor. So the output is all technically markdown, but html files contain HTML markup.

It could also make sense to hook this in to the the html exporter/importer instead. But updating that exporter/importer would make this already large PR a bit too big for my taste.